packages/irmin/irmin.0.10.1/opam

```diff
@@ -37,10 +37,9 @@ depends: [
   "cmdliner"
   "crunch"
   "base-unix" {test}
-  "git" {test}
-  "cohttp" {test}
-  "alcotest" {test & >= "0.4.1"}
-  "ocamlbuild" {build}
```

Comment:
I assume `ocamlbuild` is still necessary for building... (which you dropped here)